kazoo/client.py

```diff
@@ -294,6 +294,13 @@ def _retry(*args, **kwargs):
             raise TypeError('__init__() got unexpected keyword arguments: %s'
                             % (kwargs.keys(),))
 
+    def __enter__(self):
+        return self
+
+    def __exit__(self):
```

Comment:
Fix this guys signature.

https://docs.quantifiedcode.com/python-anti-patterns/correctness/exit_must_accept_three_arguments.html#anti-pattern